services/capture/src/main/java/ca/bc/gov/tno/services/capture/handlers/ScheduledService.java

```diff
@@ -67,14 +65,15 @@ public ScheduledService(final ServiceState state,
    * @param ranAt  The date and time the transaction ran at.
    */
   @Override
-  protected void updateDataSource(DataSourceConfig dataSource, ScheduleConfig schedule, ZonedDateTime ranOn) {
+  protected synchronized void updateDataSource(DataSourceConfig dataSource, ScheduleConfig schedule, ZonedDateTime ranOn) {
```

Comment:
What does the synchronized keywork do?

Reply:
Another merge error. This was to temporarily prevent more than one thread from executing this method. It was throwing Hibernate errors because two TransactionCompleteEvents were being generated instead of one (an issue I fixed before the demo yesterday). I had removed it before the git issue yesterday afternoon.